microservices/dms_sync/services/sync_orchestrator.py

```
@@ -0,0 +1,400 @@
+                    _server_path = Path(__file__).resolve().parent.parent.parent.parent / "server"
+                    if str(_server_path) not in sys.path:
+                        sys.path.insert(0, str(_server_path))
+                    from app.models.rooftop import Rooftop
+                    from app.models.service_record import ServiceRecord
+                    from app.models.call import Call
+
+                rooftop_result = await session.execute(
+                    select(Rooftop).where(Rooftop.id == rooftop_id)
+                )
+                rooftop = rooftop_result.scalar_one_or_none()
+                rooftop_name = rooftop.name if rooftop else f"Rooftop {rooftop_id}"
```

Ruff (0.14.14)

[error] Local variable `rooftop_name` is assigned to but never used

Remove assignment to unused variable `rooftop_name`

(F841)